sema/check/__main__.py

```
@@ -0,0 +1,72 @@
+
+    return parser
+
+
+def make_service(args: Namespace) -> Check:
+    return Check(
+        input_folder=args.input_folder,
+        output=args.output,
+    )
+
+
+def _main(*args_list) -> bool:
```

Ruff

Missing type annotation for `*args_list`

(ANN002)